<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderStatus</class>
 <widget class="QDialog" name="RenderStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>7</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="simLab">
       <property name="text">
        <string>Simulation progress: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="displayLab">
       <property name="text">
        <string>Display progress: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="renderLab">
       <property name="text">
        <string>Render progress: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QProgressBar" name="simProgress">
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="displayProgress">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QProgressBar" name="renderProgress">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
